src/containers/Viewer/ViewerDialog.tsx

```diff
@@ -4,16 +4,19 @@ import { closeViewer } from '@state/viewer'
 import { useEffect } from 'react'
 import Viewer from './Viewer'
 import styled from 'styled-components'
-import { useLocation } from 'react-router'
 import { $Any } from '@/types'
 import isHTMLElement from '@helpers/isHTMLElement'
+import clsx from 'clsx'
 
 const StyledDialog = styled(Dialog)`
   /* dnd overlay must offset this 64px by 32px */
   width: calc(100% - 64px);
   height: calc(99% - 64px);
   max-height: unset;
   max-width: unset;
+  &.isHidden {
```

Comment:
Purpose to this?

Reply:
Leftover from initial attempts, removing it.